packages/monitoring/tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "outDir": "./dist",
     "declarationDir": "./dist",
     "resolveJsonModule": true,
-    "sourceMap": true
```

Comment:
rollup was complaining about module needing to be set to `Node16` 

This didn't make sense as we had a chain of extends going to `@tsconfig/node22/tsconfig.json` which sets it there. 

This was preventing typechecking entirely

Looks like there's a [bug](https://github.com/rollup/plugins/issues/1583) where extending doesn't work